explorer/app/components/Layout/Layout.tsx

```diff
@@ -1,24 +1,19 @@
+// Core dependencies
 import { Box } from "@mui/material";
-import React from "react";
+import React, { ReactNode } from "react";
 
-interface LayoutProps {
-  mainColumn: React.ReactNode;
-  sideColumn: React.ReactNode;
+interface Props {
+  mainColumn: ReactNode;
+  sideColumn: ReactNode;
+  top: ReactNode;
 }
 
-export const Layout = ({
-  mainColumn,
-  sideColumn,
-}: LayoutProps): JSX.Element => {
+export const Layout = ({ mainColumn, sideColumn, top }: Props): JSX.Element => {
   return (
-    <Box
-      display="flex"
-      flexDirection="row"
-      width="100%"
-      gap={(theme) => theme.spacing(4)}
-    >
-      <Box width="100%">{mainColumn}</Box>
-      <Box width={350} display="flex">
+    <Box color="ink" display="grid" gap={4} mx="auto" maxWidth={1232}>
```

Comment:
I see you have made a couple of layout changes here and wanted to check on the status of the general layout. For example, the project title is currently jammed up against the header; should this be fixed here or when you take a look at [Update detail page layout components to match refined mocks](https://github.com/clevercanary/data-browser/issues/102)?